src/Compilers/CSharp/Portable/Symbols/Synthesized/GeneratedNames.cs

```diff
@@ -393,9 +393,9 @@ internal static string MakeSynthesizedLocalName(SynthesizedLocalKind kind, ref i
             return null;
         }
 
-        internal static string MakeSynthesizedInstrumentationPayloadFieldName(MethodSymbol method, int methodOrdinal)
+        internal static string MakeSynthesizedInstrumentationPayloadLocalName(int uniqueId)
```

Comment:
I'd keep it "Field" instead of "Local" since the result will always be a field of a display class. Only user-defined locals and locals that legacy EE recognized by names need to have names.